### ISSUE TITLE
[0.F stable] Bike racked vehicle sometimes fused with carrying vehicle

### ISSUE THREAD

**Issue by PatrikLundell** [CleverRaven/Cataclysm-DDA#50215]
### Describe the bug

I've found a electric car with a bike rack, as well as 3 single tile vehicles in a junkyard, and tried to bring those vehicles "home". Suspecting there might be issues with side-to-side carried vehicles, I saved before starting, which was good, as one of them had fused when I returned.

I've tried to repeat the problem, and managed to get a carried vehicle to fuse about twice more, but failed to actually reproduce it in a repeatable manner. In one of the cases I'd loaded all 3 vehicles, and one was stuck, while in another I'd loaded only two vehicles and they were NOT adjacent to each other, but one stuck nevertheless.

The attached save is from after returning to "base", this time successfully being able to repeat the issue.

### Steps To Reproduce

1. 'e'xamine to the west
2. Load or unload vehicle -> You load the Welding Cart on the rack [why isn't the handjack listed as an alternative?]
3. 'e'xamine to the west
4. Load or unload vehicle
5. Load a vehicle on the rack
6. 'e'xamine to the west
7. Remove the Handjack from the rack
8. move behind the handjack, grab it, pull it away and then push it back to verify it's detached (probably not significant)
9. Move back and 'e'xamine to the west
10. Load or unload a vehicle
11. Nothing happens, i.e. you exit the menu when selecting the option above.
12. Move to outside of the Welding Cart, grab it, and see "You grab the Electric SUV with Bike Rack", rather than the cart.

### Expected behavior

Loaded vehicles should detach when unloaded.

### Versions and configuration

- OS: Windows
    - OS Version: 10.0 2009
- Game Version: bc28921 [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    Bionic Professions [package_bionic_professions]
]

### Additional context

[Auburn Hills - Copy.zip](https://github.com/CleverRaven/Cataclysm-DDA/files/6874127/Auburn.Hills.-.Copy.zip)

The save is from after returning back to "base" after hauling the vehicles successfully. I've then hauled the detached vehicles back to adjacent to the vehicle before saving.

Looking at the steps, I suspect the original hauling might already have corrupted something, given the missing choice in step 2.

Prior to hauling the 3 single tile vehicles back, I've use the rack only once, and that was to haul an MC back.

I hope vehicle parts indices aren't used, as they're not safe to use as references (if they are, adding/removing vehicle parts while vehicles are loaded might also screw things up: shaving a side mirror off against an obstacle, for instance)...

**Comment by ZhilkinSerg** [comment]
Probably a duplicate of https://github.com/CleverRaven/Cataclysm-DDA/issues/48154

**Comment by actual-nh** [comment]
Ping: @Jamuro-g?

**Comment by wapcaplet** [comment]
Closing as duplicate of #48154, #52005
